daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/dfa/Rules.scala

```diff
@@ -104,7 +106,9 @@ abstract class State(states: => ArrayBuffer[State]) extends Serializable {
     val res =
       if (pTerm0.isInstanceOf[WSPStarState]) {
         val wspStar = pTerm0.asInstanceOf[WSPStarState]
-        if (wspStar.checkMatch(charIn)) { true } // Was a space
+        if (wspStar.checkMatch(charIn)) {
+          true
+        } // Was a space
```

Comment:
Comment should be moved to previous line and push previous line below it, e.g.,

```scala
          // Was a space.
          //
          true
```

This makes this branch look similar to the following branches and therefore makes the entire expression easier to read.